wdtk-client/src/test/java/org/wikidata/wdtk/client/JsonSerializationActionTest.java

```diff
@@ -60,6 +60,7 @@ public void testDefaults() {
 		assertTrue(action instanceof JsonSerializationAction);
 		assertFalse(action.needsSites());
 		assertTrue(action.isReady());
+		assertEquals(action.getActionName(), "JsonSerializationAction");
 	}
 
```

Comment:
I see, so it's another file to test. But why can't you test the method in the Client? If your code is really untestable, then it should be changed to be testable (at least I don't see any function here that cannot be tested in principle). Indeed, with the code in the client, you could even create a mock action that creates a predefined report so that you can test the exact string. But I agree that this test should be part of the client test file, not of this action's test file.